fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -88,20 +90,23 @@ class GridSearch(ReductionsEstimator):
     _MESSAGE_X_SENSITIVE_ROWS = "X and the sensitive features must have same number of rows"
 
     def __init__(self,
-                 learner,
+                 estimator,
                  disparity_metric,
-                 quality_metric,
+                 selection_rule=_TRADEOFF_OPTIMIZATION,
```

Comment:
If it's a default value, the constant shouldn't be notionally private (i.e. start with underscore)

Reply:
I was thinking about "tradeoff_optimization" as something that the users should be inputting rather than grid_search.TRADEOFF_OPTIMIZATION, that's why the underscore, but I can drop the underscore.